keyboards/toffee_studio/blueberry/config.h

```diff
@@ -0,0 +1,49 @@
+/* Copyright 2023 Toffee Studio
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+/* Keyboard Matrix Assignments */
+#define MATRIX_ROW_PINS { E6, B0, B1, F6, F5, F1, F7, F0, F4 }
+#define MATRIX_COL_PINS { B6, D4, D6, D7, B4, B5, C6, C7 }
+
+#define RGB_DI_PIN D0
+#ifdef RGB_DI_PIN
+#   define RGBLED_NUM 22
+
+/* RGB LED logical order map */
+/* Top->Bottom, Right->Left */
+#define RGBLIGHT_LED_MAP {      \
```

Comment:
To be blunt, you might be better off using rgb matrix, with how this is wired up.

Reply:
its working as is 🤔 
tbh; i wasnt the one who did the fw, and dont have much experience with it either.

what would be the potential benefits from changing it?
its working as is, so i dont see a reason to change it, id also need to bother the person who did the fw in the first and ask them to spend some more time on it, even tho its already working.

Reply:
RGB Light is meant for linear effects, while rgb matrix is meant for 2D animations. 

While RGB Light works, as somebody that has boards that support layouts like this, rgb matrix produces a much better experience visually, because of how the animations are designed. 

And the config needed isn't very difficult, either.  And if needed, I can see about helping with that.

Reply:
If you dont mind; my (not toffeestudio's) discord nick is Suzu#9521
iirc tho, we had a bit of trouble with file size already due to using a 32u4; so if matrix is more complex/bigger in filesize it likely wouldnt be possible to add/change it

Reply:
RGB Matrix is more geared towards per-key LEDs. If these are underglow (on the back of the PCB), RGBlight is fine here.